src/Collections-Strings/Symbol.class.st

```diff
@@ -496,6 +496,12 @@ Symbol >> isUnary [
 	^ self precedence = 1
 ]
 
+{ #category : #comparing }
+Symbol >> literalEqual: other [
+
+	^ self class == other class and: [self == other]
```

Comment:
why not just `self == other`? It is not possible that  `self == other ` but  `self class ~~ other class`

Reply:
Good catch.